src/wasm-type.h

```diff
@@ -73,7 +82,7 @@ constexpr Type anyref = Type::anyref;
 constexpr Type exnref = Type::exnref;
 constexpr Type unreachable = Type::unreachable;
 
-const char* printType(Type type);
+const std::string printType(Type type);
```

Comment:
does this change have an impact on performance? we may call this many many times in a huge wast.

Reply:
I believe that in the common case performance should still be good, but I'm not entirely sure. I expect to clean up this printing to use ostreams in a follow up PR, so I don't think it's worth blocking on here.